src/components/navigation/NavigationLine.js

```diff
@@ -23,9 +26,13 @@ export default function NavigationLine() {
   const dispatch = useDispatch();
   const { pathname, hash } = useLocation();
 
+  const currentUser = useSelector(getCurrentUserById);
+
   const isTabletView = useSelector(getIsTabletView);
   const isMobileView = useSelector(getIsMobileView);
 
+  useUploadAvatarUrl();
```

Comment:
What's the idea here?

Reply:
If the current user object has an avatar object field, then make a request to load the url for this avatar.

The code is wrapped in a hook because it is used in two components.

Reply:
@Oleksandr1414 ok, then `uploadAvatarsUrls` is probably a wrong meaning, since it's not an upload, but a build and assign.

Maybe we can rename to `buildAndSetAvatarsUrls`.

Reply:
@Oleksandr1414 I'm also not sure this is a good place for this hook, since the file is related to Navigation and has nothing related to user or avatars

Reply:
The user's profile button contains their avatar.

Perhaps the best solution would be to move the hook from the two components to the authorization stage, and after authorization, upload the user's avatar.

![image](https://github.com/SAMA-Communications/sama-client/assets/98953475/2273d6e0-1425-405b-9b36-a155839bf480)